sycl/test-e2e/Graph/RecordReplay/record_inconsistent_devices.cpp

```diff
@@ -0,0 +1,38 @@
+// REQUIRES: opencl-aot, level_zero, gpu
```

Comment:
Will the CI runs for each specific backend actually have multiple GPU platforms available? If not this test may never actually be run.

Reply:
I get that, my question is more will the test ever actually be run on CI if OpenCL isn't always installed. I suppose there is still some value in having it available to run locally if multiple platforms are installed.